apps/ui/src/composables/useUserSkin.ts

```diff
@@ -1,9 +1,12 @@
 type Skin = 'dark' | 'light' | 'none';
 
+const preferredColor = usePreferredColorScheme();
 export function useUserSkin() {
```

Comment:
We should always have new line between variable and a function
```suggestion
const preferredColor = usePreferredColorScheme();

export function useUserSkin() {
```

Reply:
We have this in a few other places on this repo,  better to add a new eslint rule